doc/sphinx/source/users-guide/base/forloops.rst

```diff
@@ -0,0 +1,113 @@
+:title: Users Guide: For loops
+
+.. _ug-forloops:
+
+For-loops
+=========
+
+*For-loops* are Chapel's means of expressing serial iteration as
```

Comment:
Hi Preston: I thought about that, but fear a little bit writing a users guide that contrasts us with other things (because, in the limit, there are so many distinct languages we could contrast ourselves with).  What specifically were you thinking we should say here?